common/source/docs/common-flight-controller-wiring.rst

```diff
@@ -21,6 +21,9 @@ GPS is mandatory in all vehicles. Compass is mandatory for Copter and Rover and
 
 .. note:: Usually GPS is attached to the logical Serial Port 3 by default in ArduPilot. However, which  physical UART is assigned to ArduPilot's Serial Port 3 on the autopilot is documented in the autopilots :ref:`documentation <common-autopilots>`
 
+.. note:: While it is possible to use multiple GPSes in a system (See :ref:`<ek3-affinity-lane-switching>`), most use only one. It is important that it be connected to the first SERIALx port that has its ``SERIALx_PROTOCOL`` parameter set to "5" (GPS) since it will stop searching for GPS during bootup if not found on the first port configured for GPS protocol.
```

Comment:
I think we should change the first part to be something like:

note:: If only one GPS is used it is important that it be connected to the first SERIALx port that has its ``SERIALx_PROTOCOL`` parameter set to "5" (GPS) since it will stop searching for GPS during bootup if not found on the first port configured for GPS protocol

Reply:
thanks..done